bitnami/tomcat/values.yaml

```diff
@@ -269,3 +269,11 @@ readinessProbe:
   periodSeconds: 51
   failureThreshold: 3
   timeoutSeconds: 3
+
+## Additional labels to apply to all resources
+labels: {}
```

Comment:
Can you use `commonLabels` instead? For standardization with other charts, i.e: `bitnami/discourse`.